### PR TITLE
Docs: Add documentation for reporting.enabled config option

### PR DESCRIPTION
**Pull request by pianohacker**
**What this PR does / why we need it**:

Adds documentation for recently-added config option to disable reporting.

### CODE REVIEW

File: docs/sources/enterprise/reporting.md [hunk 1]
```diff
@@ -162,6 +162,8 @@ These options are available in the [configuration]({{< relref "../administration
 
 ```ini
 [reporting]
```
Comment (achatterjee-grafana) [inline docs/sources/enterprise/reporting.md:164]: Is this enabled by default?

**Review by achatterjee-grafana — COMMENTED** [review]
Added a question

File: docs/sources/enterprise/reporting.md [hunk 2]
```diff
@@ -162,6 +162,8 @@ These options are available in the [configuration]({{< relref "../administration
...
+# Enable/disable the reporting feature. If disabled, no reports will be generated and the UI will be hidden.
```
Comment (achatterjee-grafana) [inline docs/sources/enterprise/reporting.md:165]:
```suggestion
# Use this option to enable or disable the reporting feature. When disabled, no reports are generated, and the UI is hidden. By default, reporting is enabled?.
```

**Review by achatterjee-grafana — COMMENTED** [review]
Added doc revew.

File: docs/sources/enterprise/reporting.md (hunk 1 above)
Comment (timlevett) [reply docs/sources/enterprise/reporting.md:164]: it is

**Review by timlevett — APPROVED** [review]

**Pull request merged**